test/auth_test.rb

```diff
@@ -560,6 +552,15 @@ def app
           last_response.headers["Location"]
       end
     end
+
+    describe "for users that have previously used an SSO" do
+      it "removes the sso_entity cookie when successfully logging in" do
+        post "/login", { email:  "kerry@heroku.com", password: "abcdefgh" },
+          { "HTTP_X_FORWARDED_FOR" => "8.7.6.5", "rack.session" => { "foo" => "bar", "sso_entity" => "initech"  } }
```

Comment:
Line is too long. [115/80]<br>Align the parameters of a method call if they span more than one line.<br>Redundant curly braces around a hash parameter.